File: angular/src/app/main/finance/shared/dto/arinvh-dto.ts
import { ARINVDDto } from "./arinvd-dto";

export class ARINVHDto {
    id: number;
    docNo: number;
    docDate: Date | string | null;
    invDate: Date | string | null;
    locID: number | null;
    routID: number | null;
    refNo: number;
    saleTypeID: string;
    paymentOption: string;
    narration: string;
    bankID: string;
    accountID: string;
    configID: number | null;
    chequeNo: string;
    linkDetID: number | null;
    audtUser: string;
    audtDate: string | null;
    createdBy: string;
    createDate: Date | string | null;
    posted: boolean;
    postedBy: string;
    postedDate: Date | string | null;
    arinvDetailDto: ARINVDDto[] = new Array<ARINVDDto>();
}

File: aspnet-core/src/ERP.Application.Shared/AccountReceivables/RouteInvoices/Dtos/ARINVHDto.cs

using System;
using Abp.Application.Services.Dto;

namespace ERP.AccountReceivables.RouteInvoices.Dtos
{
    public class ARINVHDto : EntityDto
    {
		public int DocNo { get; set; }

		public DateTime? DocDate { get; set; }

		public DateTime? InvDate { get; set; }

		public int? LocID { get; set; }

		public int? RoutID { get; set; }

		public int? RefNo { get; set; }

		public string SaleTypeID { get; set; }

		public string PaymentOption { get; set; }

		public string Narration { get; set; }

		public string BankID { get; set; }

		public string AccountID { get; set; }

		public int? ConfigID { get; set; }

		public string ChequeNo { get; set; }

		public int? LinkDetID { get; set; }

		public string AudtUser { get; set; }

		public DateTime? AudtDate { get; set; }

		public string CreatedBy { get; set; }

		public DateTime? CreateDate { get; set; }

		public bool Posted { get; set; }

		public string PostedBy { get; set; }

		public DateTime? PostedDate { get; set; }

		


    }
}